docs/how-to/cooperative_groups.rst

```diff
@@ -0,0 +1,448 @@
+.. meta::
+  :description: This topic describes how to use cooperative groups in HIP
+  :keywords: AMD, ROCm, HIP, cooperative groups
+
+*******************************************************************************
+Cooperative Groups
+*******************************************************************************
+
+Cooperative groups API is an extension to the ROCm programming model. It provides developers with a flexible grouping mechanism. This feature was introduced with ROCm 4.1 at AMD platform and CUDA 9.0 at NVIDIA platform.
```

Comment:
Do we need to discuss when this feature was added. Seems unnecessary.

Reply:
Removed.